build/write-img-mac.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I'm torn on whether having a separate fork of this file for mac would be useful / maintainable. Aside from `write-img`, do you know if the other scripts even work on mac right now?

Reply:
download-image works as I recall.